src/layouts/LoginLayout/LoginLayout.spec.tsx

```diff
@@ -85,4 +85,17 @@ describe('LoginLayout', () => {
       expect(child).toBeInTheDocument()
     })
   })
+
+  describe('when session is expired', () => {
+    it('renders the expiry banner', async () => {
+      setup()
+      jest
+        .spyOn(window.localStorage.__proto__, 'getItem')
+        .mockReturnValue('true')
```

Comment:
Same thing with this mock here, can you quickly move it into the `setup` function.